docs/shared/tutorials/react-standalone.md

```diff
@@ -25,273 +24,361 @@ Note, while you could easily use Nx together with your manually set up React app
-- a Cypress based set of e2e tests (`e2e/`)
-- Prettier preconfigured
+- a new React application at the root of the repository (`src`)
 - ESLint preconfigured
-- Jest preconfigured
+- Vite preconfigured
 
-Let me explain a couple of things that might be new to you.
+You can build the application with the following command:
 
-| File           | Description                                                                                                                                                                                                          |
-| -------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `nx.json`      | This is where we fine-tune how Nx works. We define what [cacheable operations](/features/cache-task-results) there are, and configure our [task pipeline](/concepts/task-pipeline-configuration). More on that soon. |
-| `project.json` | This file is where you can modify the inferred tasks for the `myreactapp` project. More about this later.                                                                                                            |
+```
+npm run build
+```
 
-## Serving the App
+## Add Nx
 
-The most common tasks are already mapped in the `package.json` file:
+Nx offers many features, but at its core, it is a task runner. Out of the box, it can cache your tasks and ensure those tasks are run in the correct order. After the initial set up, you can incrementally add on other features that would be helpful in your organization.
 
-```json {% fileName="package.json" %}
-{
-  "name": "myreactapp",
-  "scripts": {
-    "start": "nx serve",
-    "build": "nx build",
-    "test": "nx test"
-  }
-  ...
-}
+To enable Nx in your repository, run a single command:
+
+```shell {% path="~/react-app" %}
+npx nx@latest init
 ```
 
-To serve your new React application, just run: `npm start`. Alternatively you can directly use Nx by using
+This command will download the latest version of Nx and help set up your repository to take advantage of it.
 
-```shell
-nx serve
-```
+First, the script will propose installing some plugins based on the packages that are being used in your repository.
 
-Your application should be served at [http://localhost:4200](http://localhost:4200).
+- Leave the plugins deselected so that we can explore what Nx provides without any plugins.
 
-Nx uses the following syntax to run tasks:
+Second, the script asks a series of questions to help set up caching for you.
 
-![Syntax for Running Tasks in Nx](/shared/images/run-target-syntax.svg)
+- `Which scripts are cacheable?` - Choose `build` and `lint`
+- `Does the "build" script create any outputs?` - Enter `dist`
+- `Does the "lint" script create any outputs?` - Enter nothing
+- `Would you like remote caching to make your build faster?` - Choose `Skip for now`
```

Comment:
Should we mention that we'll setup Nx Cloud to enable CI and distribution later?